src/controllers/webhookController.test.ts

```
@@ -186,7 +190,7 @@ describe("webhookController", () => {
 
     it("returns 401 when verif-hash header is absent", () => {
       const rawBody = Buffer.from("{}");
-      const req = { headers: {}, rawBody } as unknown as RawRequest;
+      const req = { headers: { "x-flw-timestamp": validTimestamp() }, rawBody } as unknown as RawRequest;
```

ESLint

[error] Replace `·headers:·{·"x-flw-timestamp":·validTimestamp()·},·rawBody` with `⏎········headers:·{·"x-flw-timestamp":·validTimestamp()·},⏎········rawBody,⏎·····`

(prettier/prettier)